flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/runtime/sort/LimitOperator.java

```diff
@@ -26,7 +26,7 @@
 import org.apache.flink.util.Collector;
 
 /**
- * Operator for limit.
+ * Operator for batch limit.
```

Comment:
Can we relocate the operators into `stream` or `batch` sub-package`? 

Currently, the classes names are quite confusing. IMO, adding a comment doesn't help a lot here.